### ISSUE TITLE
RStudio server keeps randomly crashing due to a same jsonrpc error

### ISSUE THREAD

**Issue by judy-resmed** [rstudio/rstudio#12817]
Hi team, multiple users in my team have been experiencing some random crashes, taking a look at the log I found they seem like caused by the same jsonrpc error:
```
2023-03-09T17:41:28.759247Z [rserver-monitor] ERROR 2023-03-09T17:41:28.759048Z [rsession-bend1] ERROR jsonrpc error 7 (Method not found) [method: set_computed_theme_colors]
OCCURRED AT void rstudio::session::rpc::handleRpcRequest(const rstudio::core::json::JsonRpcRequest&, rstudio_boost::shared_ptr<rstudio::session::HttpConnection>, rstudio::session::http_methods::ConnectionType) src/cpp/session/SessionRpc.cpp:345; 
LOGGED FROM: void rstudio::session::rpc::handleRpcRequest(const rstudio::core::json::JsonRpcRequest&, rstudio_boost::shared_ptr<rstudio::session::HttpConnection>, rstudio::session::http_methods::ConnectionType) src/cpp/session/SessionRpc.cpp:350\|\|\|
--
2023-03-09T17:31:00.937519Z [rserver-monitor] ERROR 2023-03-09T17:31:00.937149Z [rsession-bend1] ERROR The previous R session terminated abnormally
LOGGED FROM: rstudio::core::Error {anonymous}::rInit(const rstudio::r::session::RInitInfo&) src/cpp/session/SessionMain.cpp:721\|\|\|
```

The crashes happened randomly and usually can be fixed by restart the session, so I can't reproduce it on my end.

The RStudio we are using is 4.2.2 (2022-10-31), and we run the server on Amazon Linux 2 x86_64-pc-linux-gnu (64-bit). So far the RStudio is still mostly running fine for us, but any suggestion or help on this issue would be appreciated!

**Comment by ronblum** [comment]
@judy-resmed Thank you for raising this issue! 4.2.2 is the version of R, though. You can get the version of RStudio with
```
RStudio.Version()$long
```
in the console, or else looking at Help -> About RStudio. What value do you have for that?

**Comment by judy-resmed** [comment]
@ronblum `2022.07.2+576.pro12` is the value from running RStudio.Version()$long

**Comment by ronblum** [comment]
@judy-resmed Ah! It looks like you're using Posit Workbench, is that correct? Rather than the open-source RStudio Server?

**Comment by judy-resmed** [comment]
@ronblum yes, it's Posit Workbench

**Comment by ronblum** [comment]
@judy-resmed Can you[ file a Support ticket](https://posit.co/support/)? The support level for professional process is different and is handled by the Support engineers. Thanks!

**Comment by judy-resmed** [comment]
Hi team, the crashing issue got solved by upgrading the posit workbench to 2022.12.01 version, so i will hold off filing a support ticket until it crashes again. I assume the issue (related to the `set_computed_theme_colors`?) was in the last version and got fixed in the latest one. Please feel free to close this issue! thank you for the help.

**Comment by jeffvroom** [comment]
Yes, prior to 2022.12, the R call to get the theme colors would send an event, then wait-for the browser to send the rpc-method method every time it was called. Now that happens only on the first call. I'm not sure why it crashed before, but that approach was slow and error prone for sure.
